docroot/modules/custom/foia_webform/src/FoiaSubmissionApiService.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
```

Comment:
A lot of these have a `namespace` of `Drupal\foia_webform\Plugin\QueueWorker` but they should just be `Drupal\foia_webform`. I'd just double check the `namespace` throughout to confirm everything is correct.